quickstart/performance-tips.md

```diff
@@ -93,17 +93,16 @@ cat /sys/devices/system/cpu/isolated
 {% endtab %}
 {% endtabs %}
 
-### Deploy Mayastor daemonset
-
-Edit the `mayastor-daemonset.yaml` file and set the `-l` parameter of mayastor to specify CPU cores that Mayastor reactors should run on. In the following example we run mayastor on the third and fourth CPU core:
-
-```yaml
-  ...
-  containers:
-    - name: mayastor
-      ...
-      args:
-        ...
-        - "-l3,4"
+### Update mayastor helm chart for CPU core specification
+
+To allot specific CPU cores for Mayastor's reactors, follow these steps:
+
+1. Ensure that you have the Mayastor kubectl plugin installed, matching the version of your Mayastor Helm chart deployment ([releases](https://github.com/openebs/mayastor/releases)). You can find installation instructions in the [Mayastor kubectl plugin documentation](https://github.com/openebs/mayastor-extensions/blob/develop/k8s/plugin/README.md).
```

Comment:
Let's replace the REAME.md link with this one -- https://mayastor.gitbook.io/introduction/advanced-operations/kubectl-plugin

On second thought, that once isn't very helpful for a naive user.